src/v2/Components/Address/AddressModalFields.tsx

```diff
@@ -34,7 +34,7 @@ export const AddressModalFields: React.FC = props => {
           onChange={handleChange}
           onBlur={handleBlur}
           error={touched.name && errors.name}
-          value={values?.name}
+          value={values?.name ? values?.name : undefined}
```

Comment:
Is this supposed to return `undefined` if `values?.name` is `null`?

I think `values?.name || undefined` should work here